v03_pipeline/lib/tasks/reference_data/updated_reference_dataset_collection.py

```diff
@@ -22,6 +30,29 @@ def _destination_path(self) -> str:
             self.reference_dataset_collection,
         )
 
+    def complete(self) -> bool:
+        self._datasets_to_update.clear()
+        all_datasets_for_collection = self.reference_dataset_collection.datasets(
+            self.dataset_type,
+        )
+
+        if not self.output().exists():
+            logger.info('Creating a new reference dataset collection')
+            self._datasets_to_update.extend(all_datasets_for_collection)
+            return False
+
+        joined_ht = hl.read_table(self._destination_path)
```

Comment:
You can also you `self.output().path` instead of `_destination_path`... I think I'm not particularly opinionated on this one since I think you'll see both patterns in the tasks thus far.

Reply:
ok yeah I removed the extra `destination_path` property since it's kind of redundant